projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/ai-assistant/with-ai-assistant.tsx

```diff
@@ -14,8 +14,7 @@ import AiAssistantDropdown, {
 	AiAssistantDropdownOnChangeOptionsArgProps,
 	KEY_ASK_AI_ASSISTANT,
 } from '../../components/ai-assistant-controls';
-import useTextContentFromSelectedBlocks from '../../hooks/use-text-content-from-selected-blocks';
```

Comment:
Is `useTextContentFromSelectedBlocks` still needed? Can it be removed? I think we should try to remove it so we don't reintroduce the perf problem.

Reply:
I don't think this is resolved? I don't see this function removed